orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -419,6 +419,13 @@ def check_function_has_param(function: Callable, param_name: str) -> bool:
     return param_name in inspect.signature(function).parameters
 
 
+def get_fully_qualified_class_name(klass):
```

Comment:
```suggestion
def get_fully_qualified_class_name(klass) -> str:
```

I would add types to all newly defined functions